client/css/reviewRating.scss

```diff
@@ -0,0 +1,114 @@
+.reviewRatingContainer{
+  border:1px solid black;
+  padding: 10px;
+}
+.reviewList{
+  width: 60%;
+  margin: auto;
+  padding: 10px;
+}
+
+.review{
+  margin:20px auto;
+  border-bottom: 1px solid black;
+  padding-bottom: 10px;
+}
+.rating_reviewer_date{
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 5px;
+}
+.summary{
+  font-weight:bold;
+  margin-bottom: 5px;
+}
+.reviewBody{
+  margin-bottom: 5px;
+}
+.reviewBody img{
+  margin:20px 10px;
+}
+.recommend{
+  margin-bottom: 10px;
+  display: flex;
+  align-items: center;
+}
+.reviewResponse{
+  margin-bottom: 5px;
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(235, 228, 228);
+  padding: 5px 10px;
+}
+.reviewResponse p{
+  margin: 5px;
+}
+.helpful span{
+  margin:5px 10px;
+}
+
+.modal{
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 10;
+  background-color: rgb(212, 235, 233);
+  border:1px solid black;
+  height: fit-content;
+  max-width: 60%;
+  min-width: 30%;
+  padding:5px;
+  max-height: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 10px 5px 5px grey;
+  border-radius: 12px;
+  object-fit: contain;
+}
+.modal .inputLabel{
+  width:100%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  margin:5px auto;
+
+}
+.formInput{
+  width: 50%;
+}
+
+.characteristicsLabel div{
+  display: flex;
+  justify-content: space-between;
+  margin:5px auto;
+}
+.reviewBtn{
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+.reviewBtn button{
+  margin:10px 20px;
```

Comment:
There should be white space between the colon and the property value.

Reply:
Yes, I wonder if we can install CSS formatter to a quick fix.

Reply:
I bet one exists, but it will also be able important to see and type code that doesn't look quite right stylistically. It will become second nature after a while.